src/renderers/shared/fiber/ReactFiberClassComponent.js

```diff
@@ -47,19 +46,19 @@ module.exports = function(scheduleUpdate : (fiber: Fiber, priorityLevel : Priori
       const updateQueue = fiber.updateQueue ?
         addToQueue(fiber.updateQueue, partialState) :
         createUpdateQueue(partialState);
-      scheduleUpdateQueue(fiber, updateQueue, LowPriority);
+      scheduleUpdateQueue(fiber, updateQueue);
```

Comment:
I think I found the problem... If `scheduleUpdateQueue` is sync (not batched) then the update is flushed before `enqueueCallback` is ever called. Then `enqueueCallback` will add a callback to the queue but it won't ever get called until the next time the fiber is committed.